packages/playground/blueprints/src/lib/steps/rmdir.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { NodePHP } from '@php-wasm/node';
+import { RecommendedPHPVersion } from '@wp-playground/wordpress';
+import { rmdir } from './rmdir';
+
+const docroot = '/php';
+describe('Blueprint step rmdir()', () => {
+	let php: NodePHP;
+	beforeEach(async () => {
+		php = await NodePHP.load(RecommendedPHPVersion);
+		php.mkdir(docroot);
+	});
+
+	it('should remove a directory', async () => {
+		php.mkdir(`/${docroot}/dir1`);
+		await rmdir(php, {
+			path: `/${docroot}/dir1`,
+		});
+		expect(php.fileExists(`/${docroot}/dir1`)).toBe(false);
```

Comment:
`/${docroot}` resolves to `//php` – note the double slash.

You could do this instead:

```ts
import { joinPaths } from '@php-wasm/util';
joinPaths( docroot, 'dir1' );
```

Reply:
Alternatively, you could also hardcode `/php` in each of these tests, it should be a bit more readable that way while still easy to change via mass search&replace.

Reply:
Thanks for the remarks. I wholeheartedly agree and will take care of that.

However, I see another issue that this PR might fix. The scheme in the PHP Blueprint library is slightly altered - this step and rm step will work differently. To accommodate this - I would advise to make the rmdir step call internally the rm step (to preserve the API for the time being), and the rm step to call `remove()`. This will align their behavior with the future php lib sheme.

There is also the issue of how offensively the new lib will handle invalid paths. Should this also happen here? This is a potentially breaking change, since invalid paths that did nothing would now throw exceptions.

Reply:
There are two migration paths to choose from:

1. Quietly rewrite v1 Blueprints as v2 Blueprints and pass them on to the new library. There won't be 1:1 parity in nuanced error handling, but perhaps that's fine.
2. Keep the TypeScript library to handle v1 Blueprints, only address the most critical vulnerabilities, recommend everyone to migrate to v2.

I'm not sure which one we'll take yet.

Reply:
That is to say there's no need to eagerly address a potential future BC issues at this point because the problem isn't well defined yet.

Reply:
Yeah. There's a lot to consider. In this particular case I would make the adjustment without the offensive error handling (which would be breaking). But making` rm` and `rmdir` make the same thing internally poses no real issues, since one is a subset of the other. A small change that could make v2 feel very similar, when the time to migrate comes.

Reply:
> A small change that could make v2 feel very similar, when the time to migrate comes.

You know, as in choosing a same-price car with better gas-mileage. You can afford the extra gas expense, but why would you if you don't have to?

Reply:
`rm` accepts files while `rmdir` accepts directories, they're not a subset of each other in this implementation. Changing that would be a change in the API and would require considering the consequences for the Blueprint developers. I don't see good reasons to do it now. If it's useful later, let's do it later.